catroid/src/androidTest/java/org/catrobat/catroid/uitest/ui/fragment/LookFragmentTest.java

```diff
@@ -807,9 +964,25 @@ public void testDragAndDropDown() {
 		assertEquals("Wrong List after DragAndDropTest", lookDataList.get(2).getLookName(), "TestLook1");
 		assertEquals("Wrong List after DragAndDropTest", lookDataList.get(3).getLookName(), SECOND_TEST_LOOK_NAME);
 		assertEquals("Wrong List after DragAndDropTest", lookDataList.get(4).getLookName(), "TestLook2");
+
+		undo();
+
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(0).getLookName(), FIRST_TEST_LOOK_NAME);
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(1).getLookName(), SECOND_TEST_LOOK_NAME);
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(2).getLookName(), "TestLook0");
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(3).getLookName(), "TestLook1");
+		assertEquals("Wrong List before DragAndDropTest", lookDataList.get(4).getLookName(), "TestLook2");
```

Comment:
As you can see above, the looks are created dynamically and not static with fixed names. I think it is fine that way, because those looks are never used in any of the other testcases, because you should never need that much. If I really should change to static names, please give a good reason to do so ;D